articles/modules/ROOT/pages/what-is-pattern-comprehension-and-why-should-i-care.adoc

```diff
@@ -0,0 +1,78 @@
+= What is pattern comprehension and why should I care?
+:slug: what-is-pattern-comprehension-and-why-should-i-care
+:author: Sven
+:category: cypher
+:tags: cypher, OGM, SDN, Mapping
+:neo4j-versions: 3.5, 4.x
+
+
+See this documentation section for a short introduction to "Pattern Comprehension":
+
+https://neo4j.com/docs/cypher-manual/current/syntax/lists/#cypher-pattern-comprehension
+
+In some situations, it is beneficial to use the concept of pattern comprehension over the OPTIONAL MATCH statements.
```

Comment:
`the concept of pattern comprehension` might be a little wordy. Just `pattern comprehensions` should be fine.